megamek/src/megamek/server/DedicatedServer.java

```diff
@@ -67,21 +65,13 @@ public static void start(String[] args) {
 
         try {
             server = new Server(resolver.password, resolver.port, new GameManager(), resolver.registerServer, resolver.announceUrl, mailer, true);
-            MegaMek.printToOut(Messages.getFormattedString("MegaMek.ServerStarted", server.getHost(), server.getPort(), server.isPassworded() ? "enabled" : "disabled") + "\n");
         } catch (Exception ex) {
             LogManager.getLogger().error("Error: could not start server at localhost" + ":" + resolver.port, ex);
-            MegaMek.printToOut(Messages.getFormattedString("MegaMek.ServerStartFailed"));
-            MegaMek.printToOut(ex.getLocalizedMessage());
             return;
         }
 
         if (null != resolver.saveGameFileName) {
             server.loadGame(new File(resolver.saveGameFileName));
         }
     }
-
-    public static void main(String[] args) {
```

Comment:
Are we uh, still able to start a dedicated server?

Reply:
Yes, this was just a manual dev use starter... that hasn't worked in years (since Log4j2 was installed at the latest).